lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -60,7 +60,12 @@ local function split(s, sep, plain, opts)
   opts = opts or {}
   local t = {}
   for c in vim.gsplit(s, sep, plain) do
-    table.insert(t, c)
+    local line = opts.file_encoding and vim.iconv(c, opts.file_encoding, "utf8") or c
+    if line ~= nil then
```

Comment:
i prefer something like `line = line or c` and then do the insert.

Reply:
Actually, going back to the handling `nil` thing, if `vim.iconv` returns `nil` wouldn't the above "ternary operator" handle it and just return `c`.
We should be able to simply `table.insert(t, line)`.